src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -100,7 +100,11 @@ static async Task RunAsync(ServerConfiguration serverConfiguration, Cancellation
     // TODO: Remove, the path should match exactly. Workaround for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1830914.
     Microsoft.CodeAnalysis.EditAndContinue.EditAndContinueMethodDebugInfoReader.IgnoreCaseWhenComparingDocumentNames = Path.DirectorySeparatorChar == '\\';
 
-    var server = new LanguageServerHost(Console.OpenStandardInput(), Console.OpenStandardOutput(), exportProvider, loggerFactory.CreateLogger(nameof(LanguageServerHost)));
+    // Named pipe server is actually created from the client, so here we create a client stream.
+    var pipeServer = new NamedPipeClientStream(".", GetDotNetPipeConnectionString(serverConfiguration.PipeName), PipeDirection.InOut, PipeOptions.Asynchronous);
```

Comment:
what is the `"."` for?

Reply:
Machine name, since they can be cross-machine I think. That said, a named parameter would be good here!

Reply:
As Jason said in another comment, this is the machine name and "." indicates this machine. I've added the named parameter, let me know if a comment would also be helpful.